app/src/main/java/com/javahelp/frontend/domain/user/register/RegisterResult.java

```diff
@@ -0,0 +1,66 @@
+package com.javahelp.frontend.domain.user.register;
```

Comment:
I feel like `com.javahelp.backend.domain.user.register.UserRegisterResult` could be reused here, and further extended like set `Token`, etc.

Reply:
Hmmm I don't think so Lam because the frontend cannot depend on the backend so they cannot share the same class.